src/data/compositors/kwin.json

```
@@ -1,6 +1,6 @@
 {
-  "generationTimestamp": 1721830546465,
-  "version": "6.1",
+  "generationTimestamp": 1730641507766,
+  "version": "6.2.2",
```

Some time ago we decided to drop the patch numbers from the string, as we doubt anyone adds new protocols in patch releases: https://github.com/vially/wayland-explorer/pull/68

```suggestion
  "version": "6.2",
```